src/ray/core_worker/common.h

```diff
@@ -168,4 +168,22 @@ struct PlacementGroupCreationOptions {
   const bool is_detached = false;
 };
 
+struct ObjectLocation {
```

Comment:
I found it might not easy to directly use rpc::GetObjectLocationsOwnerReply (a protobuf object) for cython as the type of GetObjectLocationsOwnerReply.node_ids is `RepeatedField`, but I could be wrong